src/main/java/org/venice/beachfront/bfapi/geoserver/GeoserverEnvironment.java

```diff
@@ -48,6 +48,8 @@
 	private String DATASTORE_NAME;
 	@Value("${geoserver.layer.name}")
 	private String LAYER_NAME;
+    @Value("${geoserver.layergroup.name}")
+    private String LAYER_GROUP_NAME;
```

Comment:
Just make sure those spaces line up.  Not sure if it's spaces vs tabs or something

Reply:
All non-tab indents should be fixed now.